Quaver.Shared/Screens/Gameplay/Rulesets/Keys/HitObjects/HitObjectManagerKeys.cs

```diff
@@ -38,7 +38,7 @@ public static float ScrollSpeed
             get
             {
                 var speed = MapManager.Selected.Value.Qua.Mode == GameMode.Keys4 ? ConfigManager.ScrollSpeed4K : ConfigManager.ScrollSpeed7K;
-                return speed.Value / (20f * AudioEngine.Track.Rate);
+                return speed.Value / (20f * AudioEngine.Track.Rate) * 1920 / 1366;
```

Comment:
Would be good to have the scale somewhere global

Reply:
Yea, where would a good place be?

Reply:
The `QuaverGame` class should be fine